package J007_exception;

public class E001_ExceptionDemo {
	public static void main(String[] args) {
		
		System.out.println("Program started....");
		
		try 
		{
			int a = 10;
			int b = a/10;
			System.out.println(b);
		} 
		catch (ArithmeticException e)
		{
			//e.printStackTrace();
		}
	
		System.out.println("Program endned...");
	}
}
